Modules/_csv.c

```diff
@@ -131,7 +131,7 @@ typedef struct {
     Py_UCS4 *field;             /* temporary buffer */
     Py_ssize_t field_size;      /* size of allocated buffer */
     Py_ssize_t field_len;       /* length of current field */
-    int numeric_field;          /* treat field as numeric */
+    bool unquoted_field;
```

Comment:
All other lines have a little comment, could you add one here?

Reply:
(I have disabled auto-merge to await your reaction to these, hope that wasn’t rude!)

Reply:
Thank you for doing this.
```suggestion
    bool unquoted_field;        /* true if no quotes around the current field */
```